### PR TITLE
AppNotificationList: Improves position now that topnav is live

### PR DESCRIPTION
**Pull request by torkelo**
* Rewrite to functional component 
* Move styles from sass to emotion
* Improve position 

Todo
* take topnav collapsed state into account 

Screenshot from 2023-04-20 15-17-04

### CODE REVIEW

File: public/app/core/components/AppNotifications/AppNotificationList.tsx [hunk 1]
```diff
@@ -16,53 +16,48 @@ import {
...
+function getStyles(theme: GrafanaTheme2) {
+  return {
+    wrapper: css({
+      label: 'app-notifications-list',
+      zIndex: theme.zIndex.portal,
+      minWidth: 400,
+      maxWidth: 600,
+      position: 'fixed',
+      right: 10,
+      top: 60,
```
Comment (ashharrison90) [inline public/app/core/components/AppNotifications/AppNotificationList.tsx:60]: isn't this just the same position as before? 🤔

**Review by ashharrison90 — COMMENTED** [review]
functional component refactor looks good 👍 

question about the positioning change (or not? 😕 )

**Comment by torkelo** [comment]
@ashharrison90 ah, probably did not push my latest change :)

**Review by ashharrison90 — APPROVED** [review]
lgtm 👍

**Pull request merged**